### DISCUSSION TITLE
Code Interpreter执行画图时无法显示

### DISCUSSION THREAD

**Discussion by zhaoqi9** [zai-org/ChatGLM3#1076] [Q&A]
![111](https://github.com/THUDM/ChatGLM3/assets/25588431/350acd00-11e0-47be-8f36-dae577e19cd2)
提示如下错误：
/tmp/ipykernel_500201/2005418999.py:14: UserWarning: FigureCanvasAgg is non-interactive, and thus cannot be shown
  plt.show()

但是我提问数学问题，它可以调用python执行输出答案，唯独画图无法显示，大家有遇到吗？

**Comment by Wangwl93** [comment]
您好，这个问题解决了吗，我也遇到了同样的问题

**Comment by chaorenq** [comment]
![9FDFFECFE28A52495C71C307D6F38EB3](https://github.com/THUDM/ChatGLM3/assets/156528269/64635786-148a-46af-a3fe-89d0762281dd)
我代码图像也无法显示，求指导

**Comment by xzxzlala** [comment]
delete these import in demo_ci.py works for me
```
import streamlit as st
from streamlit.delta_generator import DeltaGenerator
from client import get_client
from conversation import postprocess_text, preprocess_text, Conversation, Role
```
